src/ert/storage/__init__.py

```diff
@@ -244,18 +327,19 @@ def export_field(
 
     def export_field_many(
         self,
-        config_node: FieldConfig,
+        key: str,
         realizations: List[int],
         output_path: str,
         fformat: str,
     ) -> None:
         for realization in realizations:
             file_name = output_path % realization
             try:
-                self.export_field(config_node, realization, file_name, fformat)
-                print(f"{config_node.get_key()}[{realization:03d}] -> {file_name}")
+                self.export_field(key, realization, file_name, fformat)
+                print(f"{key}[{realization:03d}] -> {file_name}")
```

Comment:
Could you create an issue to convert this to logging? Or just convert it to logging perhaps

Reply:
I think this print is needed by Semeio. See #4649